src/ProjectTemplates/Web.ProjectTemplates/RazorClassLibrary-CSharp.csproj.in

```diff
@@ -1,12 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework Condition="'$(NoPagesAndViews)' != 'True'">netcoreapp3.0</TargetFramework>
+    <TargetFramework Condition="'$(NoPagesAndViews)' == 'True'">netstandard2.0</TargetFramework>
```

Comment:
If we are defaulting to the non-MVC case, then the name of this condition could be inverted (e.g., `SupportPagesAndViews`) so as to leave the default value as `false`.

@danroth27 Are we changing the default now, or do you anticipate us changing it later?

Reply:
Oh sorry, I see that was already addressed in comments elsewhere. As in, we are defaulting to the MVC case.

**However** if we are planning to change it later, it would make sense to pick the name now that we plan to stick with, e.g., `SupportPagesAndViews`. We can configure the default to `true` in `template.json`, and then later on that's the only place we'd have to change it.

This option name is visible to developers if they are using the CLI templates, so we should pick the name we want in the long term.

Reply:
> Oh sorry, I see that was already addressed in comments elsewhere. As in, we are defaulting to the MVC case.
> 
> **However** if we are planning to change it later, it would make sense to pick the name now that we plan to stick with, e.g., `SupportPagesAndViews`. We can configure the default to `true` in `template.json`, and then later on that's the only place we'd have to change it.
> 
> This option name is visible to developers if they are using the CLI templates, so we should pick the name we want in the long term.

I see that you can simply do -switchOpt (true|false) and be explicit about it, so I'll revert it and simply change the default

Reply:
I've updated the values. For reference:

`dotnet new razorclasslib -o ..\..\bin\tests\rcldefaultstotrue`

**rcldefaultstotrue.csproj**

```xml
<Project Sdk="Microsoft.NET.Sdk.Razor">

  <PropertyGroup>
    <TargetFramework>netcoreapp3.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <FrameworkReference Include="Microsoft.AspNetCore.App" />
  </ItemGroup>

</Project>
```

`dotnet new razorclasslib -s false -o ..\..\bin\tests\rclpassfalse`

**rclpassfalse.csproj**

```xml
<Project Sdk="Microsoft.NET.Sdk.Razor">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.0.0-preview3-t000" />
    <PackageReference Include="Microsoft.AspNetCore.Components.Build" Version="3.0.0-preview3-t000" PrivateAssets="all" />
  </ItemGroup>

</Project>
```